vars/main-srsran.yml

```
@@ -0,0 +1,88 @@
+k8s:
+  rke2:
+    version: v1.23.4+rke2r1
+    config:
+      token: purdue-k8s-rke2
+      port: 9345
+      params_file:
+        master: "deps/k8s/roles/rke2/templates/master-config.yaml"
+        worker: "deps/k8s/roles/rke2/templates/worker-config.yaml"
+
+  kubectl:
+    version: v1.23.15
+
+  helm:
+    version: v3.10.3
+
+core:
+  standalone: true		# set to false to place under control of ROC
+  data_iface: ens18
+  values_file: "deps/5gc/roles/core/templates/sdcore-5g-values.yaml"
+  ran_subnet: "172.20.0.0/16"	# set to empty string to get subnet from 'data_iface'
+  helm:
+    local_charts: false		# set chart_ref to local path name if true
+    chart_ref: aether/sd-core
+    chart_version: 1.1.0
+  upf:
+    ip_prefix: "192.168.252.0/24"
+    iface: "access"
+    mode: af_packet		# Options: af_packet or dpdk
+    default_upf:
+      ip:
+        access: "192.168.252.3/24"
+        core:   "192.168.250.3/24"
+      ue_ip_pool: "172.250.0.0/16"
+  amf:
+    ip: "10.76.28.113"
+
+srsran:
+  docker:
+    container:
+      gnb_image: aetherproject/srsran-gnb:0.0.1-dev
+      ue_image: aetherproject/srsran-ue:0.0.1-dev
```

Now we have "official" release images
```suggestion
      gnb_image: aetherproject/srsran-gnb:rel-0.0.1
      ue_image: aetherproject/srsran-ue:rel-0.0.1
```

---

done